docs/configuration.md

```diff
@@ -34,6 +34,18 @@ This command actually uses [the template file](../templates/cluster-template.yam
 
 Note: You can use [the template file](../templates/cluster-template.yaml) by manually replacing values.
 
+Note: If you wish to use cluster template without load-balancer, use command
```

Comment:
I want to care external cloud provider as well.

Suggestion:
By default the command creates highly available control plane with internal OpenStack cloud provider. If you wish to create highly available control plane with external OpenStack cloud provider or single control plane without load balancer, use external-cloud-provider or without-lb [flavor](https://cluster-api.sigs.k8s.io/clusterctl/commands/config-cluster.html#flavors) respectively.

Reply:
I think we should say following is a sample, not mandatory
they should adjust the param such as `18.2` and `3` below

Reply:
One thing tends to misconfigure is control plane count is not 1 for non ha template.

Reply:
@hidekazuna Great, Adding external provider template reference too.

Reply:
@jichenjc Sorry I didnt get you.
You would not want example to explicitly mention k8s version, count ...?

Reply:
no , I am only saying we need emphasis it's a `example` and they need change it according to their env

Reply:
@jichenjc Have mentioned the commands to be as examples in the new commit.